### PR TITLE
add FAQ for linux use

### PR DESCRIPTION
**Pull request by 20kdc**
This is bare-essentials stuff to get TestGame running, may not cover all requirements, it's just there so that it's easier for someone who doesn't know what's going on with the whole CEF stuff to get things running.

### CODE REVIEW

File: LINUX_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+*Some other information (written by Vera, PJB alerted me to this):* https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Client.CEF/CefManager.cs#L52
+
+## Compile errors involving RobustToolbox stuff
+
+You probably need to update RobustToolbox, cd over there, `git pull master`, `git submodule update --init --recursive`
+
+## Known-good CEF build
+
+`https://cef-builds.spotifycdn.com/cef_binary_91.1.6%2Bg8a752eb%2Bchromium-91.0.4472.77_linux64_beta_minimal.tar.bz2`
```
Comment (ZeWaka) [inline LINUX_FAQ.md:17]: this is hillarious

Comment (20kdc) [reply LINUX_FAQ.md:17]: ...hey, it worked for me

**Comment by 20kdc** [comment]
in regards to #337, I'm subscribing to that so I know if it's merged
it doesn't really matter which fix is merged so long as the fix works

File: LINUX_FAQ.md (hunk 1 above)
Comment (PJB3005) [reply LINUX_FAQ.md:17]: It's either Spotify's CEF builds or spend-two-days-compiling-from-source. Pick one.

**Pull request merged**